src/stores/BreadcrumbsStore.ts

```diff
@@ -59,6 +60,33 @@ export class BreadcrumbsStore extends AsyncStoreWithClient<IState> {
         return this.matrixClient && this.matrixClient.getVisibleRooms().length >= 20;
     }
 
+    componentDidUpdate(prevProps, prevState) {
+        const prevRoomCount = (prevState.rooms?.length || 0);
+        const currentRoomCount = (this.state.rooms?.length || 0)
+
+        /**
+         * Only decrypting the breadcrumb rooms events on app initialisation
+         * when room count transitions from 0 to the number of rooms it contains
+         */
+        if (prevRoomCount === 0 && currentRoomCount > prevRoomCount) {
+            const client = MatrixClientPeg.get();
+            /**
+             * Rooms in the breadcrumb have a good chance to be interacted with
+             * again by a user. Decrypting the messages ahead of time will help
+             * reduce content shift on first render
+             */
+            this.state.rooms?.forEach(async room => {
+                const [cryptoEvent] = room.currentState.getStateEvents("m.room.encryption");
+                if (cryptoEvent) {
+                    if (!client.isRoomEncrypted(room.roomId)) {
+                        await client._crypto.onCryptoEvent(cryptoEvent);
```

Comment:
I thought we exposed this properly? If not, should either be exposed or acknowledged as private member access